packages/babel-jest/src/index.ts

```diff
@@ -58,18 +65,15 @@ const createTransformer = (
     return babelConfig;
   }
 
-  return {
+  const transformer: BabelJestTransformer = {
     canInstrument: true,
+    createTransformer,
```

Comment:
this was the solution to the `FIXME` below 🙂

Reply:
This broke all usages of `createTransformer` (vs using the transformer that's already created by Jest) because the object `createTransformer` returns also has a `createTransformer` property, which is checked in `@babel/transform` to use that instead of the passed one.

Reply:
Oh, oops! 😛 Good catch!